specification/containerservice/resource-manager/Microsoft.ContainerService/aks/preview/2023-09-02-preview/managedClusters.json

```
@@ -3815,6 +3815,10 @@
         "gpuProfile": {
           "$ref": "#/definitions/AgentPoolGPUProfile",
           "description": "The GPU settings of an agent pool."
+        },
+        "artifactStreamingProfile": {
+          "$ref": "#/definitions/ArtifactStreamingProfile",
```

```suggestion
          "$ref": "#/definitions/AgentPoolArtifactStreamingProfile",
```
or rename the definition below